frontend/src/libs/components/icon/common.ts

```diff
@@ -0,0 +1,8 @@
+import { ReactComponent as Arrow } from '~/assets/img/arrow-left.svg';
+import { IconName } from '~/libs/enums/enums.js';
```

Comment:
```suggestion
```
you don't need an enum for it. create an union type for it `type IconType = 'arrow'`